### ISSUE TITLE
Difference in build setup

### ISSUE THREAD

**Issue by talicopanda** [karansher/computer-graphics-raster-images#67]
When I build raster.sln, Visual Studio creates raster.exe and raster.pdb under a new Debug folder. Is there a way to prevent this behavior from happening or can I / should I just change the file path in main.cpp to ../../data/<filename>?

**Comment by abhimadan** [comment]
Please read the assignment readme - this is the expected behaviour. Running directly from Visual Studio or from the command line in the `build` folder both work.

**Comment by talicopanda** [comment]
Right, but my question is more if it is a problem to run it from the Debug folder and change the file paths from a marking standpoint. Nothing would be affected since we are only submitting our .cpp files under the `src `folder, correct?

**Comment by abhimadan** [comment]
Sure, feel free to change main.cpp if you feel that it makes things easier, but make sure everything still works with the original version.
